requirements/dev.txt

```diff
@@ -1,90 +1,101 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --output-file=requirements/dev.txt --resolver=backtracking requirements/dev.in requirements/run.txt
+#    pip-compile --output-file=requirements/dev.txt requirements/dev.in requirements/run.txt
 #
 -e .
     # via -r requirements/dev.in
 -e git+https://github.com/kytos-ng/sphinx-theme.git#egg=kytos-sphinx-theme
     # via -r requirements/dev.in
--e git+https://github.com/kytos-ng/python-openflow.git#egg=python-openflow
+-e git+https://github.com/kytos-ng/python-openflow.git@upgrade/python#egg=python-openflow
```

Comment:
Aldo, this pinned branch here also needs to be removed, otherwise after merging it'll result in other issues since this branch will eventually be deleted.